eZ/Publish/API/Repository/SectionService.php

```diff
@@ -108,7 +108,7 @@ public function isSectionUsed(Section $section);
      * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
      * @param \eZ\Publish\API\Repository\Values\Content\Section $section
      */
-    public function assignSection(ContentInfo $contentInfo, Section $section);
+    public function assignSection(ContentInfo $contentInfo, Section $section): void;
```

Comment:
If you could, please, correct in the meantime `Assigns the content to the given section this method overrides the current assigned section.` to be on one line